Library/Homebrew/cmd/audit.rb

```diff
@@ -855,6 +855,14 @@ def audit_line(line, lineno)
       if line =~ /(require ["']formula["'])/
         problem "`#{$1}` is now unnecessary"
       end
+
+      if line =~ /#\{share\}\/#{formula.name}/
+        problem "Use \#{pkgshare}\ instead of \#{share}\/#{formula.name}"
+      end
+
+      if line =~ /share\/"#{formula.name}"/
+        problem "Use pkgshare instead of (share/\"#{formula.name}\")"
```

Comment:
I think it could be confusing because that's not what you are checking for, even if what you are checking for is most likely to appear in parentheses. But maybe that's just me being anal.